lib/shoulda/matchers/util.rb

```diff
@@ -23,6 +23,11 @@ def self.safe_constantize(camel_cased_word)
           end
         end
       end
+
+      def self.indent(string, width)
+        indentation = " " * width
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.